src/executor/source/Index.ts

```diff
@@ -0,0 +1,22 @@
+import express from 'express'
```

Comment:
Why is this file name capitalized? What is the rule here?

Reply:
There isn't really a convention on file naming. I looked to React and similar frameworks where, as far as I saw, the convention is to use PascalCase.

That's why I capitalized index.ts as well, so everything is named in a similar manner.

But there isn't really a reason to use camelCase, PascalCase or kebab-case besides personal preference. So I'm open for discussion on this, but regardless of the convention we decide on, we need to enforce it properly